code/script.js

```diff
@@ -1,51 +1,170 @@
-  if (sender === 'user') {
-    chat.innerHTML += `
-      <section class="user-msg">
-        <div class="bubble user-bubble">
-          <p>${message}</p>
-        </div>
-        <img src="assets/user.png" alt="User" />  
-      </section>
-    `
-    // the else if statement checks if the sender is a bot and if that's the case it inserts an html senction inside the chat with the posted message
-  } else if (sender === 'bot') {
-    chat.innerHTML += `
-      <section class="bot-msg">
-        <img src="assets/bot.png" alt="Bot" />
-        <div class="bubble bot-bubble">
-          <p>${message}</p>
-        </div>
-      </section>
-    `
+// bot ask user: what type of food would like to order and create menu button
+const askFoodType = (userName) => {
+  showMessage(
+    `Nice to meet you ${userName}! 😁 What type of food would you like to order?`,
+    `bot`
+  );
+  form.innerHTML = `
+   <button id="pizzaBtn" value="pizza">Pizza</button>
+   <button id="sweet" value="sweet">Sweet</button>
+   <button id="saladBtn" value="salad">Salad</button>
+   `;
+
+  document.getElementById("pizzaBtn").addEventListener("click", foodOptions);
+  document.getElementById("sweet").addEventListener("click", foodOptions);
+  document.getElementById("saladBtn").addEventListener("click", foodOptions);
+};
+
+//Step 4
+// when click the menu button, show messages of user and bot
+const foodOptions = (event) => {
+  event.preventDefault();
+  const type = event.target.value;
+  showMessage(`${type}`, `user`);
+
+  showMessage(
+    `Oh so you're in the mood for ${type} choice. Select something from the menu!`,
+    `bot`
+  );
+
+  if (type === "pizza") {
+    form.innerHTML = `
+      <select id="Option">
+              <option disabled selected >👇Please choose one:</option>
+              <option value="Margerita">Margerita</option>
+              <option value="Peperion">Peperion</option>
+              <option value="Vesuvio">Vesuvio</option>
+      </select>
+      `;
+  } else if (type === "sweet") {
+    form.innerHTML = `
+    <select id="Option">
+            <option value=""  disabled selected>👇Please choose one:</option>
+            <option value="Pastries">Pastries</option>
+            <option value="Smulpaj">Smulpaj</option>
+            <option value="Kladdkaka">Kladdkaka</option>
+    </select>
+    `;
+  } else {
+    form.innerHTML = `
+    <select id="Option">
+            <option  value="" disabled selected>👇Please choose one:</option>
+            <option value="Greek Salad">Greek Salad</option>
+            <option value="Caesar Salad">Caesar Salad</option>
+            <option value="Chicken Salad">Chicken Salad</option>
+    </select>
+    `;
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
-  chat.scrollTop = chat.scrollHeight
-}
+  document.getElementById("Option").addEventListener("change", finalOption);
```

Comment:
Optimal code path to finalise the chosen option through select method to the next step finalOption